conf/machine/include/imx-base.inc

```diff
@@ -224,6 +224,8 @@ MACHINEOVERRIDES_EXTENDER_FILTER_OUT = " \
     mx8dxl \
 "
 
+MACHINEOVERRIDES_FILTERED_OUT_QA_ERROR = "%s overrides cannot be used and need convertion to use the new BSP-specific overrides. Check 'meta-freescale/scripts/convert-bsp-specific-overrides'."
```

Comment:
convertion -> conversion

Reply:
One problem with the original script from the commit is that each time you run it it appends again. Is that fixed?

Reply:
Honestly, I don't know how to fix this issue. Ideas?

Reply:
This looks like something to play with: https://stackoverflow.com/questions/4953738/how-to-globally-replace-strings-in-lines-not-starting-with-a-certain-pattern#answer-4953824

Reply:
I am unsure about increasing the complexity to allow reuse. I foresee this script being used once in a layer to do the complete conversion and never again. When someone upstream a recipe, it is their responsibility to do it during the process.

Reply:
The error message does suggest the script to fix the problem. Please update the error message and in the comment at the top of the script that the script is intended for migrating existing layers to kirkstone and is not intended to be used more than once.

Reply:
Oh, perhaps you can add a simple grep to the script and fail if it finds new overrides?

Reply:
I implemented this idea. Please review.